__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -354,72 +375,66 @@ describe('getFullHistory', () => {
-    expect(moveTx).toHaveProperty('timestamp', rawMoveTx.timestamp);
-    expect(moveTx).toHaveProperty('is_voided', false);
-    expect(moveTx.parents).toEqual(rawMoveTx.parents);
+    expect(moveTx).toMatchObject({
+      tx_id: rawMoveTx.hash,
+      version: rawMoveTx.version,
+      weight: rawMoveTx.weight,
+      timestamp: rawMoveTx.timestamp,
+      is_voided: false,
+      parents: rawMoveTx.parents,
+    });
 
     // Validating inputs
     expect(moveTx.inputs).toHaveLength(rawMoveTx.inputs.length);
     for (const inputIndex in moveTx.inputs) {
-      const inputObj = moveTx.inputs[inputIndex];
-
-      // Some attributes were correctly translated
-      expect(inputObj.index).toEqual(rawMoveTx.inputs[inputIndex].index);
-      expect(inputObj.tx_id).toEqual(rawMoveTx.inputs[inputIndex].hash);
-
-      // Decoded attributes are correct
-      expect(inputObj).toHaveProperty('token', HATHOR_TOKEN_CONFIG.uid);
-      expect(inputObj).toHaveProperty('token_data', TOKEN_DATA.HTR);
-      expect(inputObj).toHaveProperty('script');
-      expect(inputObj).toHaveProperty('value', 10);
-      expect(inputObj).toHaveProperty('decoded');
-      expect(inputObj.decoded).toHaveProperty('type', 'P2PKH');
-      expect(inputObj.decoded).toHaveProperty('address', fundDestinationAddress);
+      expect(moveTx.inputs[inputIndex]).toMatchObject({
+        // Some attributes were correctly translated
+        index: rawMoveTx.inputs[inputIndex].index,
+        tx_id: rawMoveTx.inputs[inputIndex].hash,
+
+        // Decoded attributes are correct
+        token: HATHOR_TOKEN_CONFIG.uid,
+        token_data: TOKEN_DATA.HTR,
+        script: expect.any(String),
+        value: 10,
+        decoded: { type: 'P2PKH', address: fundDestinationAddress }
+      });
     }
 
     // Validating outputs
     expect(moveTx.outputs).toHaveLength(rawMoveTx.outputs.length);
     for (const outputIndex in moveTx.outputs) {
       const outputObj = moveTx.outputs[outputIndex];
 
-      // Some attributes were correctly translated
-      expect(outputObj.value).toEqual(rawMoveTx.outputs[outputIndex].value);
-      expect(outputObj.token_data).toEqual(rawMoveTx.outputs[outputIndex].tokenData);
-
-      // Decoded attributes are correct
-      expect(outputObj).toHaveProperty('token', HATHOR_TOKEN_CONFIG.uid);
-      expect(outputObj).toHaveProperty('script');
-      expect(outputObj).toHaveProperty('decoded');
-      expect(outputObj).toHaveProperty('spent_by', null);
-      expect(outputObj).toHaveProperty('selected_as_input', false);
-      expect(outputObj.decoded).toHaveProperty('type', 'P2PKH');
-
-      if (outputObj.value === txValue) {
-        expect(outputObj.decoded).toHaveProperty('address', txDestinationAddress);
-      } else {
-        expect(outputObj.decoded).toHaveProperty('address', txChangeAddress);
-      }
+      expect(outputObj).toMatchObject({
+        // Some attributes were correctly translated
```

Comment:
Which attributes were not correctly translated?

Reply:
Re-worded this on [dc82fee](https://github.com/HathorNetwork/hathor-wallet-lib/pull/393/commits/dc82fee1edfeb1867d4e95902343c36393456b22)